easybuild/easyconfigs/n/NASM/NASM-2.15.05-GCCcore-10.2.0.eb

```
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-90.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'NASM'
+version = '2.15.05'
+
+homepage = 'https://www.nasm.us/'
+
+description = """NASM: General-purpose x86 assembler"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://www.nasm.us/pub/nasm/releasebuilds/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['34fd26c70a277a9fdd54cb5ecf389badedaf48047b269d1008fbc819b24e80bc']
```

```suggestion
checksums = ['3c4b8339e5ab54b1bcb2316101f8985a5da50a3f9e504d43fa6f35668bee2fd0']
```
The 34fd... checksums if for NASM 2.14.02.